# test parameters
std_msgs/Float32MultiArray noise_levels
std_msgs/Float32MultiArray clutter_levels
uint32 iterations
